.eslintrc.js

```diff
@@ -45,8 +45,8 @@ module.exports = {
     },
   },
   rules: {
-    // Disallow console as not enabled by eslint:recommended contrary to docs
-    "no-console": "error",
+    // Set console calls to emit warnings as not enabled by eslint:recommended contrary to docs
+    "no-console": "warn",
```

Comment:
Did you check this? It might be no-console by default in which case we can delete it?

Reply:
I just double checked, it's not enabled in eslint:recommended (https://github.com/eslint/eslint/blob/master/lib/rules/no-console.js)